assets/wormhole/resources/app.yaml

```diff
@@ -0,0 +1,154 @@
+apiVersion: bundle.gravitational.io/v2
+kind: Bundle
+metadata:
+  name: telekube
```

Comment:
Why do we need a separate app which seems to be exactly like "telekube", but with a different network provided? Wouldn't it be better to add a flag to gravity install to allow selecting network type (like we do with some other fields, e.g. storage driver)?

Or will it come later and this app is temporary?

Reply:
@klizhentas Didn't want to add more installation flags, and the way it's currently built wormhole only get's vendored / included if configured by the app maintainer.

So with that limit in place, the question is now whether we should publish this app to get.gravitational/hub as well.